client/src/main/resources/templates/js/htmlreporter.js

```diff
@@ -16,36 +16,36 @@
 $(document).ready(function () {
 
   "use strict";
```

Comment:
@sundaramrajendran Hmm. Okay. Either approach (old or new) is fine with me as long as there is consistency. That said, if we switch to `"` in javascript, I'd like to retain the use of `"` for html attributes in html markup, generated by jquery, or otherwise. This change set uses `'` for html attributes in several places where before it was `"`. Meaning, we should find those lines, and change to `\"` escaping now -- which is one of the reasons I've always leaned towards `'` instead of `"` usage in javascript -- but again, either approach is fine with me.